ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -2005,80 +1878,15 @@ describe(RuntimeConfigurationPanel.name, () => {
       dataprocConfig: defaultDataprocConfig(),
       status: RuntimeStatus.CREATING,
     });
-
+    mockUseCustomRuntime();
     component();
 
     const nextButton = screen.getByRole('button', { name: 'Next' });
     expectButtonElementDisabled(nextButton);
   });
 
-  it('should send an updateRuntime API call if runtime changes do not require a delete', async () => {
```

Comment:
I don't see an equivalent test to this - change DataProc disk doesn't require update

Reply:
Replaced with "should update runtime, and not delete, when dataproc's master disk size is increased" in react-hooks test